<launch>
    <arg name="frameID_Radar" default="ARS_548"/>
    <arg name="radarIP" default="224.0.2.2"/>
    <arg name="radarPort" default="42102"/>
    <node pkg="ars548_driver" type="ars548_driver" name="radar">
        <param name="radarIP" value="$(arg radarIP)"/>
        <param name="radarPort" value="$(arg radarPort)"/>
        <param name="frameID" value="$(arg frameID_Radar)"/>
    </node>
    
    <node pkg="rviz" type="rviz" name="Visualization" args="-d $(find ars548_driver)/rviz/RadarVisualization.rviz"/>
    <node pkg="tf2_ros" type="static_transform_publisher" name="radar_tf" args="0 0 0 0 0 0 PointCloud ARS_548"/>
</launch>
